src/middleware/tests/ldp/resource.test.js

```diff
@@ -115,6 +116,54 @@ describe('Resource CRUD operations', () => {
     });
```

Comment:
Comme c'est un PATCH et non un PUT, ça ne doit pas effacer "Paris, the place to be", mais retourner "Compiègnes, the place to be" + "Paris, the place to be".